mu/logic.py

```diff
@@ -1401,6 +1413,9 @@ def quit(self, *args, **kwargs):
                 "w": self._view.width(),
                 "h": self._view.height(),
             },
+            "pa_username": self.pa_username,
+            "pa_token": self.pa_token,
+            "pa_instance": self.pa_instance,
```

Comment:
It doesn't really matter that much, but this could be in a dict (like `window` and `envars`)? e.g.

```
"python_anywhere" {
    "username": ....
    "token": ....
    "instance": ....
}
```

Reply:
Hmm... what version do you suggest..? Current latest is 2.28. How about `>=2.0.0` ?